content/docs/command-reference/fetch.md

```diff
@@ -17,22 +17,17 @@ positional arguments:
 
 ## Description
 
-Downloads DVC-tracked files from remote storage into the cache of the project
-(without placing them in the <abbr>workspace</abbr>, like `dvc pull` would).
-This makes them available for linking (or copying) into the workspace (refer to
-[`dvc config cache.type`](/doc/command-reference/config#cache)).
+Downloads tracked data from remote storage into the <abbr>cache</abbr> of the
+project (without placing it in the <abbr>workspace</abbr> like `dvc pull`). This
+makes them available for linking (or copying) into the workspace (refer to
+`dvc config cache.type`).
 
-Without arguments, `dvc fetch` ensures that the files specified in all
-`dvc.lock` and `.dvc` files in the workspace exist in the cache. The
-`--all-branches`, `--all-tags`, and `--all-commits` options enable fetching data
-for multiple Git commits.
+> Note that this does not download
+> [metafiles](/doc/user-guide/dvc-files-and-directories). That can be done with
+> `git fetch` (if using Git).
 
-The `targets` given to this command (if any) limit what to fetch. It accepts
-paths to tracked files or directories (including paths inside tracked
-directories), `.dvc` files, and stage names (found in `dvc.yaml`).
-
-Fetching is performed automatically by `dvc pull` (when the data is not already
-in the <abbr>cache</abbr>), along with `dvc checkout`:
+Fetching is performed automatically if needed by `dvc pull`, along with
```

Comment:
`data not in cache` is better than `if needed` (explicit vs some thing that is not clear what does it exactly mean)

Reply:
Updated.